src/Controls/src/Core/PickerClosedEventArgs.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Microsoft.Maui.Controls;
+
+/// <summary>
+/// Provides event data for the event that is raised when a picker control is closed.
+/// </summary>
+public class PickerClosedEventArgs : EventArgs
```

Comment:
Let's break these out to all the separate types

- TimePickerClosed, DatePickerClosed, PickerClosed (same with open eventargs)

Look at creating a static "Empty" we can use internally as well to reduce allocations